tests/conftest.py

```diff
@@ -242,6 +254,21 @@ def random_filters(test_filters_random_with_mask: Filters) -> Filters:
     return test_filters_random_with_mask
 
 
+@pytest.fixture
+def synthetic_scar_removal_args(synthetic_scars_image: np.ndarray) -> dict:
```

Comment:
These are in essence configuration options and on line 31 the `default_config.yaml` is loaded to a fixture. Subsequent to that chunks of the configuration are subset out to their own fixtures (see lines 44-110) and changes made to any of the options where tests run with different parameters.


I think it would therefore be consistent to follow this convention and have on line 113 something like...

```
@pytest.fixture
def scarring_config(default_config: Dict) -> Dict:
    """Configurations for scar removal"""
    config = default_config["remove_scars"] 
    config["threshold_low"] = 1.5
    config["threshold_high"] = 1.8
    config["max_scar_width"] = 2
    config["min_scar_length"] = 1
    config.pop("run")
    return config
```

Reply:
Done 👍